### DISCUSSION TITLE
about on homepage

### DISCUSSION THREAD

**Discussion by Taxom** [jvde-github/AIS-catcher#69] [Ideas]
Could you add a section "about" to the home page which could be set through config. 
Thank you!

Max

**Comment by jvde-github** [comment]
Like that, also need to add a credit section anyway. Tell me more. 

You are thinking of a HTML file provided by the user or a string for a dialogbox or ...?

**Comment by Taxom** [comment]
I thought about a small text field on the main page with information that can be prescribed in the config. For example, some data about the owner, type of antenna, links to different resources. In the picture, I tried to portray what I mean.
![photo_2023-02-01_05-10-31](https://user-images.githubusercontent.com/20313019/216013981-17028630-5888-4194-a2d8-78bccc7116f6.jpg)

**Comment by jvde-github** [comment]
Clear, perfect, then ideally it is some HTML line or HTML file so you can add the mark up to include links, pictures etc.

**Comment by jvde-github** [comment]
Quick implementation. User can provide some content in Markdown format, e.g:
```
# Weclome

Hi my name is Jasper. Welcome to my station.

## Hardware

RTL-SDR
RPi4

## Software

AIS-catcher
```

Which using the command:
```
AIS-catcher -N 8100 ABOUT readme.md
```

Is shown in a new tab:
![image](https://user-images.githubusercontent.com/52420030/216173365-f5d1b201-6551-45df-9f07-0331fd60d2c2.png)
